components/pd_client/src/client.rs

```diff
@@ -325,6 +325,8 @@ impl fmt::Debug for RpcClient {
 }
 
 const LEADER_CHANGE_RETRY: usize = 10;
+// periodic request like store_heartbeat, we don't need to retry.
+const NO_RETRY: usize = 1;
```

Comment:
Will it cause the store to be disconnected easier?

Reply:
what about restricting the backoff duration instead of retry times?

Reply:
I think it's not, and it is based on the next round(10s) of store heartbeat that can be handled. if it cannot, the before version most likely also cannot be handled. 
and `disconnect` status just indicated the network is unstable or has other temporary problems, and it does not affect the scheduling behavior, so I think it's safe.

Reply:
> what about restricting the backoff duration instead of retry times?

@glorv  I may do the next PR, and this PR is to minimize the change of the request that is no need to retry.